tvb_documentation/tutorials/tutorial_0_GettingStarted.rst

```diff
@@ -4,187 +4,218 @@
+
+.. figure:: figures/Projects_Wait.png
+   :alt: Be patient, it will take a few minutes.
    :scale: 30%
 
-3. Create a user for someone else by registering a new user. 
 
-.. figure:: figures/LinkAndShare_AddNewUser.png
+7. You can now see the project on the *List of all projects*.
+
+.. figure:: figures/Projects_Done.png
+   :alt: The imported project can be seen in the list of all projects.
    :scale: 30%
 
-4. You’ll receive a notification email.
 
+8. If you wish to export a project, click on the pencil icon located left to the project name.
+
+.. figure:: figures/Projects_PencilIcon.png
+   :alt: Pencil Icon
+   :scale: 30%
 
-5. Make sure the new user has been validated. 
+   Clicking on it will open the Properties page of the project. Click on the Export Project button.
 
-.. figure:: figures/LinkAndShare_ValidateNewUser.png
+.. figure:: figures/Projects_PropertiesPage.png
+   :alt: Project Properties page
    :scale: 30%
 
-6. Login with the Admin account.
+9. If you wish to delete a project, you click on the *Delete* button and the project
+   will disappear from the list.
 
 ------------------------
 
 Importing A Connectivity
 ------------------------
 
-1. Create a project (e.g. *LinkAndShare\_a*).
-
-2. Create a second project (e.g. *LinkAndShare\_b*).
-
-3. Assuming that you are working in the first project, upload a Connectivity in
-   a zip file. This was already done but you can repeat these steps.
-
-4. Go to *Projects > Data structure*. Click on *Upload Data*. An
+1. Assuming that you are working in the previously created project, upload a Connectivity in
+   a zip file. Go to *Projects > Data structure*. Click on *Upload Data*. An
    overlay with the current supported formats will appear. 
 
-5. Select *Connectivity ZIP*.
+2. Select *Connectivity ZIP*.
 
-6. Select the file *connectivity\_regions\_96.zip* found at
-   *TVB\_Distribution/tvb\_data/*.
+3. From your downloaded tvb-data folder select the file *connectivity_96.zip* found at *tvb_data/connectivity/*.
 
-.. figure:: figures/LinkAndShare_Uploaders.png
+.. figure:: figures/ImportConnectivity_Uploaders.png
    :alt: Supported data formats.
    :scale: 30%
 
-7. Add a personalized tag to this newly created datatype (e.g.
-   *conn\_96*). 
 
-8. Save the changes.
+4. Add a personalized tag to this newly created datatype (e.g.
+   *conn\_96*).
 
-.. figure:: figures/LinkAndShare_TagDatatype.png
+5. Save the changes.
+
+.. figure:: figures/ImportConnectivity_TagDatatype.png
    :alt: Add a personalized tag.
    :scale: 50%
 
 ------------------------
 
 Link And Share
---------------
+---------------
 
 1. Select the connectivity you want to share.
 
 2. In the *metadata overlay*, go to the tab *Links*. You’ll see a list with all
-   your projects. 
+   your projects except the current one.
 
-3. Link this datatype (connectivity) with the project you’ll share (e.g.
-   *Session\_II\_ShareAndLink\_b*).
+3. Link this datatype (connectivity) with the project you’ll share.
```

Comment:
Explicitly link to SecondNewProject and at point 5 mention that the user should switch to SecondNewProject in order to see the linked dt